lib/wings/active_fedora_converter.rb

```diff
@@ -141,11 +142,19 @@ def apply_attributes_to_model(af_object)
         members = Array.wrap(converted_attrs.delete(:members))
         files = converted_attrs.delete(:files)
         af_object.attributes = converted_attrs
+        if resource.try(:lease) && af_object.reflections.include?(:lease)
+          # af_object.lease.class has the same name as resource.lease.class; however, each class has a different object_id
+          # so a type mismatch happens. the code below coerces the one object into the other
+          # TODO(alishaevn): fix this!
```

Comment:
is this TODO still accurate?

Reply:
- #6134

Reply:
Personally, I've started using `TODO`s in my WIP PRs for pre-merge reminders. If something is going to get merged in a state that needs to be addressed later, I've started using `FIXME`. I only ever use my GitHub username when leaving `NOTE`s so it's clear who the original author was. 

This flow has been working for me and is by no means convention, just a suggestion.

Reply:
mmmm. I do like that.
`TODO` has always been a tricky thing for me because I can't know for sure how it's interpreted. I put my name next to it for the same reason you put yours next to `NOTE`. but I suppose to some that may infer that I'm the one who's gonna fix it. which isn't always my intention. lol. nor is it always the intention that it's gonna be fixed in the present pr. (as evidenced by all the TODO's even in this repo. 😅 )

I like sticking with the `TODO` convention because it's one that I think the majority of people are familiar with. but maybe `TODO` means to handle this thing later while `TODO(name)` means **I'm** going to handle this thing before the pr is ready??

🤔

Reply:
Might be worth revisiting with the wider team. I simply go with what makes the most sense to me on a case-by-case basis, but things that make the most sense to me often don't make any sense to others 😅